From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Sat, 14 May 2022 00:50:05 +0900
Subject: Skip test error

Forwarded: not-needed

Avoid FTBFS error by AssertionError.
Python 3.10 in reproducible build tester (build2) failed.
---
 tests/services/test_browser.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/services/test_browser.py b/tests/services/test_browser.py
index 0afc5eb..a9f701f 100644
--- a/tests/services/test_browser.py
+++ b/tests/services/test_browser.py
@@ -1453,6 +1453,7 @@ def test_service_browser_matching():
     zc.close()
 
 
+@pytest.mark.skipif(os.environ.get("DEBIAN_TEST"), reason="disabled by Debian builder")
 @patch.object(_engine, "_CACHE_CLEANUP_INTERVAL", 0.01)
 def test_service_browser_expire_callbacks():
     """Test that the ServiceBrowser matching does not match partial names."""
